manifest.datestamps.json

```
@@ -1,40 +1,46 @@
 {
     "variables": {
-      "CurrentReleaseDateStamp": "20250513",
+      "PreviousReleaseDateStamp": "20250513",
+      "CurrentReleaseDateStamp": "20250610",
 
-      "RuntimeReleaseDateStamp": "$(CurrentReleaseDateStamp)",
-      "AspnetReleaseDateStamp": "$(CurrentReleaseDateStamp)",
-      "WcfReleaseDateStamp": "$(CurrentReleaseDateStamp)",
-      "SdkReleaseDateStamp": "$(CurrentReleaseDateStamp)",
+      "RuntimeReleaseDateStamp": "$(PreviousReleaseDateStamp)",
+      "AspnetReleaseDateStamp": "$(PreviousReleaseDateStamp)",
+      "WcfReleaseDateStamp": "$(PreviousReleaseDateStamp)",
+      "SdkReleaseDateStamp": "$(PreviousReleaseDateStamp)",
```

Since the NuGet change is happening in all SDK images.

```suggestion
      "SdkReleaseDateStamp": "$(CurrentReleaseDateStamp)",
```

---

Fixed in https://github.com/microsoft/dotnet-framework-docker/pull/1246/commits/9b9894b120e16b6df61f2326a0cccb8e06db2e79